src/lib/es2020.bigint.d.ts

```diff
@@ -1,3 +1,103 @@
+type From1To21 = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 | 14 | 15 | 16 | 17 | 18 | 19 | 20 | 21;

```

Comment:
I think you should just inline these. Moving them out here pollutes the namespace and we update these files so little that it's not really making editing or reviewing less error-prone.